data/json/items/tools.json

```diff
@@ -9019,6 +9019,24 @@
     }
   },
   {
+    "id": "washboard",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "white",
+    "name": "washboard",
+    "description": "This is a wooden washboard.  You can use it to wash filthy clothing if it's supplied with soap.  It requires 1 unit of soap per use.",
+    "price": 1000,
+    "material": "wood",
+    "weight": 90,
+    "volume": 1,
+    "to_hit": -1,
+    "max_charges": 10,
+    "initial_charges": 5,
+    "charges_per_use": 1,
+    "ammo": "soap",
+    "use_action": "WASHCLOTHES"
```

Comment:
> EDIT: Oh, the soap is ammo so it won't be that easy.

By saying that I meant that I'm okay with washboards in the game. Direct use of soap would require a considerable impact to the code (definitely out of scope of this PR), but IMHO it should be done at some point (e.g. to get rid of `TOOL_ARMOR`'s which are awkward workarounds and should become regular armors with `iuse` slots).